src/styles/blocks/stars.css

```
@@ -0,0 +1,39 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("../../images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+
+  /* margin-bottom: 4px; */
+}
+
+.stars--0 :nth-child(-n){
+  background-image: url("../../images/star-active.svg");
+}
+
+.stars--1 :nth-child(-n+1) {
+  background-image: url("../../images/star-active.svg");
+}
+
+.stars--2 :nth-child(-n+2) {
+  background-image: url("../../images/star-active.svg");
+}
+
+.stars--3 :nth-child(-n+3) {
+  background-image: url("../../images/star-active.svg");
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url("../../images/star-active.svg");
+}
+
+.stars--5 :nth-child(-n+5) {
+  background-image: url("../../images/star-active.svg");
+}
```

no need to repeat background-color for every selector
```suggestion
.stars--0 :nth-child(-n),
.stars--1 :nth-child(-n+1),
.stars--2 :nth-child(-n+2),
.stars--3 :nth-child(-n+3),
.stars--4 :nth-child(-n+4),
.stars--5 :nth-child(-n+5) {
  background-image: url("../../images/star-active.svg");
}
```